velox/exec/fuzzer/AggregationFuzzerBase.cpp

```diff
@@ -230,6 +230,13 @@ std::shared_ptr<InputGenerator> AggregationFuzzerBase::findInputGenerator(
   return nullptr;
 }
 
+std::vector<RowVectorPtr> AggregationFuzzerBase::generateEmptyTable(
+    std::vector<std::string> names,
+    std::vector<TypePtr> types) {
+  auto inputType = ROW(std::move(names), std::move(types));
+  return {vectorFuzzer_.fuzzRow(inputType, 0, false)};
```

Comment:
I ran fuzzer for a while with this change locally and come across a crash in VectorFuzzer::fuzzOffsetsAndSizes() due to a division `elementSize / size` where size == 0.
https://github.com/facebookincubator/velox/blob/a8aad60a2829fa720c1e730cf09a37140c22fea5/velox/vector/fuzzer/VectorFuzzer.cpp#L609

Could you run aggregation fuzzer to see if you can reproduce and fix this issue? Please also run fuzzer for a couple of hours to ensure this new feature works properly and doesn't break the test.

Reply:
Thanks for pointing it out, removed this function and updated the PR to set `vectorSize` in `VectorFuzzer::Options` to zero. `batch_size` is used to set this variable in fuzzer options and this seems better to test for empty inputs with a given probability, instead of using a random `batch_size` in CI config which may not test for empty inputs as frequently. Could you please share your thoughts on this?